services/upgrade-service/src/main/java/com/thinkbiganalytics/server/upgrade/KyloUpgradeApp.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * 
+ */
+package com.thinkbiganalytics.server.upgrade;
+
+/*-
+ * #%L
+ * kylo-upgrade-service
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)